configuration/rules-blockly.md

```diff
@@ -0,0 +1,157 @@
+---
+layout: documentation
+title: Rules Blockly
+---
+
+# Blockly Reference
+
+One of the core feature that OpenHAB provides is writing rules to allow specific behaviour with the home automation system. The usual way of developing rules is by coding them like described in the [Textual Rules]({{base}}/configuration/rules-dsl.html). However, this art of programming may become intimidating early on and shy away away people with few or almost no experience in programming. Therefore openHAB also provides a graphical way of writing rules which allows to put together rules in a rather visual way (even though some programming background may still help).
+
+{::options toc_levels="2..4"/}
+
+- TOC
+{:toc}
+
+{: #blockly-introduction}
+## Introduction
+
+
+The basic idea behind the visual paradigm and representation within openHAB is based on the [Google Blockly Support](https://developers.google.com/blockly) which has been integrated and which provides the basic blocks for programming like the ones on the left and the right side of the below images
+
+{: #blockly-toolbox}
+*Blockly toolbox*
+
+![blockly-toolbox-1](images/blockly/blockly-toolbox-1.png)![blockly-toolbox-2](images/blockly/blockly-toolbox-2.png)![blockly-toolbox-3](images/blockly/blockly-toolbox-3.png)
+
+
+
+All of these provide general functionality that is not specific to openHAB itself. If you want to learn more about how to use them, search for the many blockly tutorials that are available.
```

Comment:
Please change this to one sentence per line.
The website build will take core of a proper presentation of paragraphs.